### ISSUE TITLE
Panel: "Controlled visibility" example close button doesn't work

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10781]
[Functional : Panel Control] : No action is getting preformed after activating the 'Close' button either by using keyboard or by mouse.

**User Experience:**
All user will face difficulty if controls will not work as per expected. 

**Test Environment:** 
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Usage:**
1.Panel - Controlled Visibility.

**Repro Steps:**
1. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge.
2. Navigate to 'Panel' control by using Tab key and activate it.
3. Dialogue pane will be appeared.
4. Navigate to 'Controlled Visibility' variant by using Tab key and activate it.
5. Press Tab key and focus will be on 'Close' button.
6. Now verify the issue after activating it by 'Enter' key and by using mouse also.

**Actual Result:**
After activating 'Close' button, no action performed. We are not able to close that window after activating the 'Close' button. We are able to close that pane after activating the 'Close Panel' button.

**Expected Result:**
After activating the 'Close'  button, 'Controlled Panel' pane should be closed.

**Attachment:**
![No action is getting preformed after activating the 'Close' button](https://user-images.githubusercontent.com/55537167/66561963-320fa600-eb78-11e9-90d2-c557ac1c1fa1.png)
[Functional_No action is getting preformed after activating the 'Close' button either by using keyboard or by mouse..zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3711976/Functional_No.action.is.getting.preformed.after.activating.the.Close.button.either.by.using.keyboard.or.by.mouse.zip)

**Comment by ecraig12345** [comment]
This example shouldn't show a close button, since the button will never do anything. Fix is to set `hasCloseButton={false}`.
